reactor-core/src/main/java/reactor/core/scheduler/Schedulers.java

```diff
@@ -151,19 +168,41 @@ public static Scheduler fromExecutorService(ExecutorService executorService, Str
 	 * <p>
 	 * This scheduler is not restartable.
 	 *
-	 * @return default instance of a {@link Scheduler} that hosts a fixed pool of single-threaded
-	 * ExecutorService-based workers and is suited for parallel work
+	 * @return default instance of a {@link Scheduler} that dynamically creates ExecutorService-based
+	 * Workers and caches the threads, reusing them once the Workers have been shut
+	 * down
 	 */
 	public static Scheduler elastic() {
 		return cache(CACHED_ELASTIC, ELASTIC, ELASTIC_SUPPLIER);
 	}
 
+	/**
+	 * {@link Scheduler} that dynamically creates a bounded number of ExecutorService-based
+	 * Workers, reusing them once the Workers have been shut down. The underlying daemon
+	 * threads can be evicted if idle for more than {@link CappedScheduler#DEFAULT_TTL_SECONDS 60} seconds.
+	 * <p>
+	 * The maximum number of created thread pools is bounded by a {@code cap} (by default,
+	 * ten times the number of available CPU cores, see {@link #DEFAULT_CAPPED_SIZE}).
+	 * If a worker is requested while the cap is reached, a facade {@link reactor.core.scheduler.Scheduler.Worker}
+	 * is provided which will enqueue the tasks submitted to it, deferring the actual submission
+	 * of tasks until a thread-backed worker becomes available. This can thus affect initial delays of tasks.
+	 * If a task is directly submitted to the {@link Scheduler} while the cap has been reached,
+	 * then it will be rejected with a {@link RejectedExecutionException}.
+	 *
+	 *
+	 * @return a new {@link Scheduler} that dynamically create workers with an upper bound to
+	 * the number of backing threads, reuses threads and evict idle ones
+	 */
+	public static Scheduler capped() {
+		return cache(CACHED_CAPPED, CAPPED, CAPPED_SUPPLIER);
+	}
+
 	/**
 	 * {@link Scheduler} that hosts a fixed pool of single-threaded ExecutorService-based
 	 * workers and is suited for parallel work.
 	 *
 	 * @return default instance of a {@link Scheduler} that hosts a fixed pool of single-threaded
-	 * ExecutorService-based workers
+	 * ExecutorService-based workers and is suited for parallel work
```

Comment:
we have to be careful with "parallel work" tho. People tend to think parallel == multi-threaded == heavy computations, while we actually use it for IO and non-blocking stuff

Reply:
I just realigned the javadocs for `elastic` and `parallel` flavors in here, but maybe in a separate follow up we could find a better wording.